### PR TITLE
feat: overview page setup

### PR DESCRIPTION
**Pull request by howema**
### Description

info
-removed all boilerplate, js, and styles
-is now a simple "Hello World" as the index route for this site
-renders on localhost:3000 via npm run dev

### Spec

See Story: [FSA22V1-161](https://sparkbox.atlassian.net/browse/FSA22V1-161)

### Validation

* [ ] This PR has code changes, and our linters still pass.
* [ ] This PR affects production code, so it was browser tested.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all linters pass.
4. Confirm that 'Hello World' renders on localhost:3000 via npm run dev

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -1,72 +1,5 @@
...
-        >
-          Powered by
-          {' '}
-          <span className={styles.logo}>
-            <Image src="/vercel.svg" alt="Vercel Logo" width={72} height={16} />
-          </span>
-        </a>
-      </footer>
-    </div>
+    'Hello World!'
```
Comment (dustin-jw) [inline pages/index.jsx:3]: I'd like to see a little more html structure here so when we add stuff in later it's a little easier.

```jsx
    <div>
      <Head>
      ...
      </Head>
      <main>
        <h1>Hello World!</h1>
      </main>
    </div>
```

**Review by dustin-jw — CHANGES_REQUESTED** [review]
We can leave a little bit more boilerplate in here than you removed 😄 This is also going to have failing tests, since the elements that `__tests__/index.test.js` is looking for won't be there. It will be much more obvious when tests fail once #4 is merged.

File: pages/index.jsx (hunk 1 above)
Comment (howema) [reply pages/index.jsx]: Hey, I made these changes. Re-requesting review :)

File: pages/index.jsx [hunk 2]
```diff
@@ -1,72 +1,5 @@
...
-import styles from '../styles/Home.module.css';
-
 export default function Home() {
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
```
[the remark this answers is not in the document]
Comment (howema) [reply pages/index.jsx]: All done!

**Review by dustin-jw — APPROVED** [review]

**Comment by dustin-jw** [comment]
@arnest00 I choose you to review!

**Review by arnest00 — APPROVED** [review]
You're good to go, @howema!

**Pull request merged**